.github/workflows/file_verification.yml

```diff
@@ -0,0 +1,62 @@
+name: Lance File Write Read Round Trip Verification
+on: [pull_request]
```

Comment:
I will make this workflow run on nightly in a following up PR, I need it to run on `pull_request` for now to make sure this workflow passes.

Reply:
We need to change before we commit because otherwise this will fail PR checks

Reply:
I actually don't understand why this will fail PR checks yet.
fixed, changed to run on `workflow_dispatch`